README.md

```diff
@@ -142,6 +142,15 @@ Note that for non-root `template` elements, if there aren't any valid directives
 
 As a corrective measure, this mod will therefore remove any `template` elements without valid directives.
 
+##### HTML parsing errors
+
+Fixes errors generated during LWC HTML parsing.
+
+The following transforms are available for each errors:
+- `eof-in-element-that-can-contain-only-text`
+- `end-tag-without-matching-open-element`
+- `closing-of-element-with-open-child-elements`
+
```

Comment:
This section is nested under the `html-template-cleanup` code mod and just explains the different fixes. 

I wanted to put examples here but these aren't official HTML errors.

I think they're specific to `parse5` and I couldn't find great examples for them other than in this [repo](https://github.com/rehypejs/rehype/blob/main/packages/rehype-parse/readme.md).

Reply:
Yeah parse5 gets some of these names from the spec, but other ones are just invented.